src/modules/commander/HealthAndArmingChecks/checks/systemCheck.cpp

```diff
@@ -121,27 +121,6 @@ void SystemChecks::checkAndReport(const Context &context, Report &reporter)
 		}
 	}
 
-	// avoidance system
-	if (context.status().avoidance_system_required) {
-		if (context.status().avoidance_system_valid) {
-			reporter.setIsPresent(health_component_t::avoidance);
```

Comment:
Should we remove the health component too?

Reply:
Yes we should - thanks for already taking care of it.

Reply:
@bkueng , Matthias and me weren't clear on how to remove this component cleanly. I propose something in https://github.com/PX4/PX4-Autopilot/pull/24172/commits/a94968e31c171e4bd2b01c45d8415685d3a57ff4 - could you have a look please?